app/models/ability.rb

```diff
@@ -24,6 +24,8 @@ def initialize(user)
       # TODO: Should all logged in users be able to view editorial pages?
       can :view, :editorial_page
 
+      can :create, Invite
```

Comment:
Should all existing users be able to invite others? Or should this be an admin function.